services/worker/poetry.lock

```diff
@@ -1735,9 +1709,9 @@ develop = true
 
 [package.dependencies]
 appdirs = "^1.4.4"
-datasets = {version = "^2.11.0", extras = ["audio", "vision"]}
+datasets = {version = "^2.12.0", extras = ["audio", "vision"]}
```

Comment:
why is that? was it missing from https://github.com/huggingface/datasets-server/pull/1147/files?

(it's not very important, I just wonder what occurred)

Reply:
This is the commit that should have updated that file (after having updated libcommon before): https://github.com/huggingface/datasets-server/pull/1147/commits/2d907bffc099000ff1365c2fdc1042df072459ca

Reply:
I ran `poetry update datasets` and maybe I should have run differently, because the dependency on datasets is not direct but indirect through libcommon.

Maybe should I have run instead `poetry update libcommon`?

Reply:
See my comment below, related to this.